README.md

```diff
@@ -2,8 +2,7 @@
 
 # Stuart Dambi Developer Profile
 
-> This repository contains the starting lessons about Gitflow
-> and Github actions first setup
+> This repository contains my developer profile
 
 ## Built With
 
```

Comment:
Good Job for adding a README file, but make sure to include enough description into your README file, and also keep and update other sections from the Microverse README file template.